modules/data.land/R/clip_and_move_raster_file.R

```diff
@@ -0,0 +1,47 @@
+#' Clip and Move a Raster File
+#'
+#' Clips a raster to a polygon bounding box, optionally masks to polygon, and saves the 
+#' output in the same format as the input.
+#'
+#' @param input_path Character. Path to the input raster file.
+#' @param polygon An `sf` or `SpatVector` object to be used for clipping and masking.
+#' @param out_path Character. Path to save the processed raster.
+#' @return Character. The path to the saved output raster.
```

Comment:
Note that the return value is identical to a required input, so it's not very informative -- would it make sense to instead return either a logical success/failure or (maybe invisibly) the clipped raster)?